src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -965,26 +965,25 @@ def test_sql_db_short_term_retention(self):
             'server_name': 'lillian-westus2-server',
             'database_name': 'ps5691',
             'retention_days_v1': 7,
-            'diffbackup_hours_v1': 24,
             'retention_days_v2': 6,
-            'diffbackup_hours_v2': 12
+            'diffbackup_hours_v2': 12,
+            'retention_days_v3': 5,
+            'diffbackup_hours_v3': 10
         })
 
-        # Test UPDATE short term retention policy on live database, value updated to v1.
+        # Test UPDATE short term retention policy (retention days only), value equals to v1.
         self.cmd(
-            'sql db str-policy set -g {resource_group} -s {server_name} -n {database_name} --retention-days {retention_days_v1} --diffbackup-hours {diffbackup_hours_v1}',
+            'sql db str-policy set -g {resource_group} -s {server_name} -n {database_name} --retention-days {retention_days_v1}',
             checks=[
                 self.check('resourceGroup', '{resource_group}'),
-                self.check('retentionDays', '{retention_days_v1}'),
-                self.check('diffBackupIntervalInHours', '{diffbackup_hours_v1}')])
+                self.check('retentionDays', '{retention_days_v1}')])
 
-        # Test GET short term retention policy on live database, value equals to v1.
+        # Test GET short term retention policy (retention days only) on live database, value equals to v1.
         self.cmd(
             'sql db str-policy show -g {resource_group} -s {server_name} -n {database_name}',
             checks=[
                 self.check('resourceGroup', '{resource_group}'),
-                self.check('retentionDays', '{retention_days_v1}'),
-                self.check('diffBackupIntervalInHours', '{diffbackup_hours_v1}')])
```

Comment:
Do you want to keep this check?

without updating diffbackup, do we still default to 24 hours?

Reply:
Let me call you to explain.